static/app/components/profiling/flamegraph/flamegraphTooltip.tsx

```diff
@@ -82,6 +82,9 @@ export function FlamegraphTooltip(props: FlamegraphTooltipProps) {
             {t('source')}:{formatFileNameAndLineColumn(props.frame)}
           </Fragment>
         )}
+        <FlamegraphTooltipTimelineInfo>
+          {props.frame.frame.is_application ? 'application' : 'system'} frame
```

Comment:
Missing the translation and we'd want to translate the whole phrase together
```suggestion
          {props.frame.frame.is_application ? t('application frame') : t('system frame')}
```